tool/icon_generator/lib/common.dart

```diff
@@ -0,0 +1 @@
+X:/Coding/Garages/Flutter/flutter_tools_metadata/tool/common.dart
```

Comment:
Was this file included deliberately, or accidentally?

Reply:
It's been removed

Reply:
Ah, I must've clicked through from the email to just one change. It all looks good to me now.